### DISCUSSION TITLE
Spacy V3 French transformer pipeline (NER)

### DISCUSSION THREAD

**Discussion by wahid18benz** [explosion/spaCy#7562] [Help: Best practices]
Hello Everyone,

I have seen that there is no **NER pipeline** component for **fr_deep_news_trf** **(camembert-base).**

Please I would like to know if there is any way to create the NER pipleine for this model ?

Thank you

**Comment by polm** [comment]
You should be able to train the NER component using the [normal training process](https://spacy.io/usage/training#quickstart). The French models are trained on the [Sequoia corpus](https://github.com/UniversalDependencies/UD_French-Sequoia), which doesn't have NER annotations, and [WikiNER](https://figshare.com/articles/dataset/Learning_multilingual_named_entity_recognition_from_Wikipedia/5462500), which does.
